### PR TITLE
Add support for EFR32MG21 Routers

### PR DESCRIPTION
**Pull request by digiblur**
Based on firmware https://github.com/xsp1989/zigbeeFirmware

### CODE REVIEW

File: devices/silabs.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const tz = require('zigbee-herdsman-converters/converters/toZigbee');
+const reporting = require('zigbee-herdsman-converters/lib/reporting');
+const extend = require('zigbee-herdsman-converters/lib/extend');
+const e = exposes.presets;
+
+module.exports = [
+    {
+        zigbeeModel: ['UT-02'],
+        model: 'EFR32MG21',
+        vendor: 'Custom devices (DiY)',
```
Comment (Koenkk) [inline devices/silabs.js:12]: Please change the vendor to `Sillicon Labs`. Also make a pr for https://github.com/koenkk/zigbee2mqtt.io and explain what hardware can be used/firmware to flash.

**Comment by digiblur** [comment]
Will see what I can migrate over to a page from these

https://www.digiblur.com/2022/02/convert-sonoff-zigbee-bridge-to-router-repeater.html
https://github.com/xsp1989/zigbeeFirmware/tree/master/firmware/Zigbee3.0_Dongle/RouterForDongle

**Comment by digiblur** [comment]
Instructions for users - https://www.digiblur.com/2022/02/convert-sonoff-zigbee-bridge-to-router-repeater.html

**Comment by Koenkk** [comment]
Could you create a pr for https://github.com/koenkk/zigbee2mqtt.io ?

**Pull request closed without merging**